src/lib/button-manager.ts

```diff
@@ -77,9 +77,19 @@ export class ButtonManager {
 
   async mount(element: Element): Promise<void> {
     if (!this.isGooglePayLoaded()) {
-      await loadScript('https://pay.google.com/gp/p/js/pay.js');
+      try {
+        await loadScript('https://pay.google.com/gp/p/js/pay.js');
+      } catch(err) {
+        if (this.config?.onError) {
+          this.config.onError(err as Error);
+        } else {
+          console.error(err);
+        }
+      }
     }
 
+  
```

Comment:
Let's also bring this back to one space separation by removing the two new lines added.